platforms/web/lib/useListeners/useListeners.ts

```diff
@@ -185,6 +199,7 @@ export function useListeners(
             editorNode.removeEventListener('paste', onPaste);
             editorNode.removeEventListener('wysiwygInput', onWysiwygInput);
             editorNode.removeEventListener('keydown', onKeyDown);
+            editorNode.removeEventListener('beforeinput', onBeforeInput);
```

Comment:
These lines create a reference to the `onPaste` function which will be called whenever a `beforeInput` event occurs. The `addEventListener` call attaches that function when the composer is created, the `removeEventListener` call removes it when the composer is removed from the page